closure/compiler/closure_js_library.bzl

```diff
@@ -16,9 +16,11 @@
 
 load(
     "//closure/private:defs.bzl",
-    "CLOSURE_JS_TOOLCHAIN_ATTRS",
+    "CLOSURE_LIBRARY_BASE_ATTR",
+    "CLOSURE_WORKER_ATTR",
     "JS_FILE_TYPE",
     "JS_LANGUAGE_IN",
+    "UNUSABLE_TYPE_DEFINITION",
```

Comment:
I added some text in `unusable_type.js`.

The ClosureCompiler adds a new annotation `@type {UnusableType}` in some places in ijs: https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/ijs/JsdocUtil.java

To avoid unrecognized type errors, I essentially copied what the google3 `js_library` rule does to define `UnusableType`.